condensation-forum/application.py

```diff
@@ -61,22 +61,23 @@ def indexGetHandler():
     """
     Returns the template "home" wrapped by "body" served as HTML
     """
+    threads = None
+    with dataSessionMgr.session_scope() as dbSession:
+        threads = query.getThreadsByCommentTime(dbSession)
+        users = [thread.user for thread in threads]
+        threads = query.extractOutput(threads)
+        users = query.extractOutput(users)
+
+
+    homeRendered = homeTemplate.render(threads_zip_users=zip(threads, users))
 
-    homeRendered = homeTemplate.render()
     user = authManager.getUserData()
     if user == None:
         homeRendered += "<br/><br/>User is not logged in"
     else:
         homeRendered += "<br/><br/>User is: " + user['name']
 
-    with dataSessionMgr.session_scope() as dbSession:
-        threads = query.getThreadsByCommentTime(dbSession)
-        if threads:
-            for thread in threads:
-                homeRendered += "<p><b>%s</b> %s</p>" % (thread.heading, thread.user.name) 
-        else:
-            homeRendered += "<p>No threads found</p>"
-    return bodyTemplate.render(body = homeRendered, title = "Test Home", user = user)
+    return bodyTemplate.render(body = homeRendered, user = user )
```

Comment:
Any reason the page title is removed? Typically to support browser bookmarking and so-on, a page title is formatted like "Website Name - Information about the page content" and this would need to be provided per route.

Reply:
I think that was an accident, I added it back in on the next pull request